ericus123/PORTFOLIO_UI
# Static Pages Metatags

#### What does this PR do?
Add static metatags

#### Description of Task to be completed?

- Check if all static pages have their metatags


#### How should this be manually tested?

How should this be manually tested?
- Visit the app url and you should see metatags of each page

#### Any background context you want to provide?
N/A
#### What are the relevant trello/pivotal tracker stories?
N/A
#### Screenshots (if appropriate)

#### Questions:

N/A
